PlayFabClientSDK/source/PlayFabHttp/IPlayFabHttp.cs

```diff
@@ -3,8 +3,8 @@
 
 namespace PlayFab.Internal
 {
-    public interface IPlayFabHttp
+    public interface IPlayFabHttp: ITransportPlugin
     {
         Task<object> DoPost(string urlPath, PlayFabRequestCommon request, string authType, string authKey, Dictionary<string, string> extraHeaders);
```

Comment:
Re comment below, I don't think we should have a second DoPost signature.
We should only have the one generic DoPost signature, and completely remove the PlayFab specific one.